### ISSUE TITLE
Typing doesn't work with `npm:@deno/kv`

### ISSUE THREAD

**Issue by RuiNtD** [oliver-oloughlin/kvdex#185]
Okay, so if you use [`npm:@deno/kv`](https://www.npmjs.com/package/@deno/kv) in Deno, the NPM package is set up to redirect to `Deno.kv` if it is defined, making it viable to use `npm:@dneo/kv` in Deno. However, when I pass a `Kv` instance from `npm:@deno/kv` into kvdex in Deno, I get a type error:

```
> deno check src/lib/db.ts
Check file:///.../src/lib/db.ts
error: TS2345 [ERROR]: Argument of type 'import("file:///.../node_modules/.deno/@deno+kv@0.7.0/node_modules/@deno/kv/esm/kv_types.d.ts").Kv' is not assignable to parameter of type 'Deno.Kv'.
  Property 'commitVersionstamp' is missing in type 'import("file:///.../node_modules/.deno/@deno+kv@0.7.0/node_modules/@deno/kv/esm/kv_types.d.ts").Kv' but required in type 'Deno.Kv'.
export const db = kvdex(kv, {
                        ~~
    at file:///.../src/lib/db.ts:28:25

    'commitVersionstamp' is declared here.
        commitVersionstamp(): symbol;
        ~~~~~~~~~~~~~~~~~~
        at asset:///lib.deno.unstable.d.ts:1940:5
```

I assume this is simply because the NPM package is more of a stable release than `Deno.kv`. A possible solution is to just change all instances of the `Deno.Kv` type to `{ Kv } from "npm:@deno/kv"`

**Comment by oliver-oloughlin** [comment]
I have been experimenting with the npm package to see if I could figure out compatibility with Node, and wasn't sure how to do it. This is a really good suggestion, I will see if I can fix this issue and make it work.

**Comment by oliver-oloughlin** [comment]
Sadly at this time, it seems the types from the NPM package and the native Deno.Kv types aren't consistent enough to actually make it cross-platform compatible. I tried moving over to using the types from npm:@deno/kv, but this still caused issues when trying to use a Deno.Kv instance with kvdex. As long as this is the case I will be prioritizing native compatibility, and hopefully we will see an improvement on this front in the future with a stable release of Deno KV.

**Comment by oliver-oloughlin** [comment]
This should now have been fixed as of #205 and release 1.0.0.
Both the native Deno.Kv object and KV from @deno/kv should be compatible.
